### PR TITLE
feat : add login page

### PR DESCRIPTION
**Pull request by Kevin-Solomon**
# Login Page

image
[Login Page CodeSandbox Link](https://u4px6.csb.app/pages/login/login.html)

### CODE REVIEW

**Review by mohammed-sas — COMMENTED** [review]
simple and clean ui.
suggestion : try different variant of white for background to improve contrast

**Review by RahullRawat — APPROVED** [review]

File: pages/login/login.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  background-color: var(--primary-color);
+}
+.navbar .right-nav {
+  display: inline-flex;
+  margin: 0;
+}
+.navbar .navbar-btn {
+  display: none;
+}
+@media (max-width: 768px) {
```
Comment (SatwikaKacham) [inline pages/login/login.css:15]: it would be better if u write media queries at the end of the CSS file. because nothing can override it.

**Review by SatwikaKacham — APPROVED** [review]
clean and neat work. good to merge

File: pages/login/login.css (hunk 1 above)
Comment (Kevin-Solomon) [reply pages/login/login.css:15]: Done

**Pull request merged**